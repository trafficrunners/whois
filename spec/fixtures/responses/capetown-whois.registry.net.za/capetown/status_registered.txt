Domain Name: registry.capetown
Domain ID: dom_3K3-9999
WHOIS Server: capetown-whois2.registry.net.za

Updated Date: 2015-05-30T09:21:00Z
Creation Date: 2015-04-01T07:41:59Z
Registry Expiry Date: 2016-04-01T07:41:59Z
Sponsoring Registrar: Lexsynergy
Sponsoring Registrar IANA ID: 1466 
Domain Status: clientDeleteProhibited
Domain Status:clientTransferProhibited
Domain Status:clientUpdateProhibited

Registrant ID: LEX-7IC-235J
Registrant Name: Lucky Mokgabudi Masilela
Registrant Organization: ZA Central Registry
Registrant Street: COZA House, Gazelle Close Corporate Park South 
Registrant City: Midrand
Registrant State/Province: Gauteng
Registrant Postal Code: 1685
Registrant Country: ZA
Registrant Phone: +27.113140077
Registrant Phone Ext: 
Registrant Fax: +27.113140088
Registrant Fax Ext: 
Registrant Email: legal@co.za

Admin ID: LEX-7IC-235J
Admin Name: Lucky Mokgabudi Masilela
Admin Organization: ZA Central Registry
Admin Street: COZA House, Gazelle Close Corporate Park South 
Admin City: Midrand
Admin State/Province: Gauteng
Admin Postal Code: 1685
Admin Country: ZA
Admin Phone: +27.113140077
Admin Phone Ext: 
Admin Fax: +27.113140088
Admin Fax Ext: 
Admin Email: legal@co.za

Billing ID: LEX-1-1XMT
Billing Name: Domain Name Department
Billing Organization: Lexsynergy Limited
Billing Street: 130 Hampstead House 176 Finchley Road 
Billing City: London
Billing State/Province: 
Billing Postal Code: NW3 6BT
Billing Country: GB
Billing Phone: +44.2081331319
Billing Phone Ext: 
Billing Fax: +44.2081331319
Billing Fax Ext: 
Billing Email: domains@lexsynergy.com

Tech ID: LEX-7IC-235J
Tech Name: Lucky Mokgabudi Masilela
Tech Organization: ZA Central Registry
Tech Street: COZA House, Gazelle Close Corporate Park South 
Tech City: Midrand
Tech State/Province: Gauteng
Tech Postal Code: 1685
Tech Country: ZA
Tech Phone: +27.113140077
Tech Phone Ext: 
Tech Fax: +27.113140088
Tech Fax Ext: 
Tech Email: legal@co.za


Name Server: ns1.nic.capetown
Name Server: ns1.dnservices.co.za
DNSSEC: unsigned
>>> Last update of WHOIS database: 2015-06-01T14:44:31Z <<<

#    WHOIS lookup made at 2015-06-01T14:44:31Z
# --
# The use of this Whois facility is subject to the following terms and
# conditions. https://registry.net.za/whois_terms
# Copyright (c) ZACR 1995-2015
