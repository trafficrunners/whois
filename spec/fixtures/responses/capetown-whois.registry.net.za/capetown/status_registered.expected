#domain
  %s == "registry.capetown"

#domain_id
  %s == "dom_3K3-9999"


#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2015-04-01 07:41:59 UTC}

#updated_on
  %s %CLASS{time}
  %s %TIME{2015-05-30 09:21:00 UTC}

#expires_on
  %s %CLASS{time}
  %s %TIME{2016-04-01 07:41:59 UTC}


#registrar
  %s %CLASS{registrar}
  %s.id           == "1466"
  %s.name         == "Lexsynergy"
  %s.organization == "Lexsynergy"
  %s.url          == nil

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_REGISTRANT
  %s[0].id            == "LEX-7IC-235J"
  %s[0].name          == "Lucky Mokgabudi Masilela"
  %s[0].organization  == "ZA Central Registry"
  %s[0].address       == "COZA House, Gazelle Close Corporate Park South"
  %s[0].city          == "Midrand"
  %s[0].zip           == "1685"
  %s[0].country       == nil
  %s[0].country_code  == "ZA"
  %s[0].phone         == "+27.113140077"
  %s[0].fax           == "+27.113140088"
  %s[0].email         == "legal@co.za"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == "LEX-7IC-235J"
  %s[0].name          == "Lucky Mokgabudi Masilela"
  %s[0].organization  == "ZA Central Registry"
  %s[0].address       == "COZA House, Gazelle Close Corporate Park South"
  %s[0].city          == "Midrand"
  %s[0].zip           == "1685"
  %s[0].country       == nil
  %s[0].country_code  == "ZA"
  %s[0].phone         == "+27.113140077"
  %s[0].fax           == "+27.113140088"
  %s[0].email         == "legal@co.za"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_TECHNICAL
  %s[0].id            == "LEX-7IC-235J"
  %s[0].name          == "Lucky Mokgabudi Masilela"
  %s[0].organization  == "ZA Central Registry"
  %s[0].address       == "COZA House, Gazelle Close Corporate Park South"
  %s[0].city          == "Midrand"
  %s[0].zip           == "1685"
  %s[0].country       == nil
  %s[0].country_code  == "ZA"
  %s[0].phone         == "+27.113140077"
  %s[0].fax           == "+27.113140088"
  %s[0].email         == "legal@co.za"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil


#nameservers
  %s %CLASS{array}
  %s %SIZE{2}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.nic.capetown"
  %s[0].ipv4 == nil
  %s[0].ipv6 == nil
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns1.dnservices.co.za"
  %s[1].ipv4 == nil
  %s[1].ipv6 == nil
